thrust/thrust/iterator/detail/tuple_of_iterator_references.h

```diff
@@ -67,7 +67,9 @@ class tuple_of_iterator_references : public thrust::tuple<Ts...>
   using super_t = thrust::tuple<Ts...>;
   using super_t::super_t;
 
-  tuple_of_iterator_references() = default;
+  inline _CCCL_HOST_DEVICE tuple_of_iterator_references()
+      : super_t()
+  {}
```

Comment:
Question: why do we need to call the base class ctor explicitly? `tuple_of_iterator_references() = default;` should do?

Reply:
I dont remember exactly why, but there were some instances, where certain compilers complained about deleted default constructors 🤷

Reply:
Do we know these compilers? Can we add a comment about this?

Reply:
I'd rather not, its not really helping and running full CI just to find that some old nvcc with some old host compiler is broken does not really add value

Reply:
So at what point in time can I remove this workaround then? This is the value of that comment. I will see this line at a brainless point in the future and drop it, because it is semantically unnecessary. A comment can tell me to keep it because of X until Y.

In any case, no change needed then. I may just find it out myself at some point.